CRM/Member/Form/MembershipRenewal.php

```diff
@@ -716,4 +742,101 @@ protected function submit() {
     }
   }
 
+  private function submitAddContributionLineItemsToParams() {
+    //create line items
+    $lineItem = array();
+    $this->_params = $this->setPriceSetParameters($this->_params);
+    CRM_Price_BAO_PriceSet::processAmount($this->_priceSet['fields'], $this->_params, $lineItem[$this->_priceSetId], NULL, $this->_priceSetId);
+
+    // litespeedmarc: commented out, this only made sense for price set based renewals
```

Comment:
@eileenmcnaughton I believe that is the correct behaviour. @litespeedmarc you don't need to respond to this comment.